module/userProjects/helper.js

```diff
@@ -6,7 +6,7 @@
  */
 
 // Dependencies
-const libraryCategoriesHelper = require(MODULES_BASE_PATH + '/library/categories/helper')
+// const libraryCategoriesHelper = require(MODULES_BASE_PATH + '/library/categories/helper')
```

Comment:
@Sachintechjoomla This has been used inside functions. Please add correct helper path